src/ui/navigator/navigationBar/VPNBottomNavigationBar.qml

```diff
@@ -160,6 +161,14 @@ Rectangle {
         }
     }
 
+    function setNavBarOpacity() {
+        if (VPNNavigator.screen === VPNNavigator.ScreenHome) {
+            navbar.opacity = VPNConnectionBenchmark.state === VPNConnectionBenchmark.StateInitial ? 1 : 0
```

Comment:
Haven't actually tested but I believe the navbar will still have 0 opacity if you start a connection benchmark and quickly click the home nav button again because we aren't actually stopping the benchmark, just reloading ScreenHome

Reply:
very very **very** good point. thank you for catching. updated.